src/main/java/org/folio/utils/EmailOkapiClient.java

```diff
@@ -18,12 +18,7 @@ public EmailOkapiClient(String okapiUrl, Vertx vertx, Map<String, String> header
 
   public void sendEmail(String url, String data) {
     post(url, data)
-      .onSuccess(response -> {
-        logger.info("POST {} complete successfully: {}", url, response);
-        close();})
-      .onFailure(t -> {
-        logger.error("Email not delivered {}", t.getMessage());
-        close();
```

Comment:
The close() method no longer exists in okapiClient.class. https://github.com/folio-org/okapi/commit/03a1ec20e95e5296fb108652d385cbf4c2e354fb
This breaking change introduced in this ticket and recommended to delete close() method 
https://issues.folio.org/browse/OKAPI-1083 
![image](https://github.com/folio-org/mod-finance-storage/assets/113523904/60028cc3-a5d6-4c92-833d-fe62cffbf380)